recipe-server/client/control_new/components/tables/ColumnSelector.js

```diff
@@ -0,0 +1,26 @@
+import { Button, Checkbox, Dropdown } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function ColumnSelector({ columns, onChange, options }) {
```

Comment:
This seems generic enough that we could abstract/rename it to something else, like `CheckboxMenu` or something. We'd just need to pass in a `label` prop or something to display the `Columns` text.